docs/plugins/transform-class-properties.md

```diff
@@ -6,6 +6,93 @@ permalink: /docs/plugins/transform-class-properties/
 package: babel-plugin-transform-class-properties
 ---
 
+Allows to use class properties
```

Comment:
Maybe change all of the headings to say something like: `This plugin allows Babel to transform x`